plugin/forward/forward.go

```diff
@@ -185,3 +189,5 @@ const (
 	randomPolicy policy = iota
 	roundRobinPolicy
 )
+
+const queryTimeout = 5 * time.Second
```

Comment:
`defaultTimeout` ?

Reply:
I can change to defaultTimeout if you wish. But this name tell us less about purpose of the constant